### PR TITLE
avoid network calls to fetch exchange rate in UI thread

### PR DESCRIPTION
**Pull request by itswisdomagain**

### CODE REVIEW

**Comment by itswisdomagain** [comment]
@ukane-philemon please check if there are use cases where network calls is okay but are currently prevented, or cases where they should be prevented but are allowed in this PR.

File: ui/page/exchange/create_order_page.go [hunk 1]
```diff
@@ -968,7 +968,7 @@ func (pg *CreateOrderPage) layoutDesktop(gtx C) D {
 									return txt.Layout(gtx)
 								}),
 								layout.Rigid(func(gtx C) D {
-									ticker := pg.AssetsManager.RateSource.GetTicker(fromCur + ext.MktSep + toCur)
+									ticker := pg.AssetsManager.RateSource.GetTicker(fromCur+ext.MktSep+toCur, false)
```
Comment (ukane-philemon) [inline ui/page/exchange/create_order_page.go:971]: Set to `true`?

**Review by ukane-philemon — CHANGES_REQUESTED** [review]
I think all `GetTicker` calls in any `Layout` method should be `cacheOnly`

File: ui/page/exchange/create_order_page.go (hunk 1 above)
Comment (itswisdomagain) [reply ui/page/exchange/create_order_page.go:971]: Oh crap. I agree should be true for all layout calls. Missed this.

Comment (itswisdomagain) [reply ui/page/exchange/create_order_page.go]: Resolved: https://github.com/crypto-power/cryptopower/compare/640545d487270723acf32e5043d4f093046630e6..86dc2534b54125c852a15abca5c73acb049951f8

**Review by ukane-philemon — APPROVED** [review]
Looks good.

PS: Failing CI.

**Review by dreacot — APPROVED** [review]

File: ui/page/accounts/accounts_page.go [hunk 2]
```diff
@@ -116,7 +116,7 @@ func (pg *Page) fetchExchangeRate() {
 		return
 	}
 
-	rate := pg.AssetsManager.RateSource.GetTicker(market)
+	rate := pg.AssetsManager.RateSource.GetTicker(market, false) // okay to fetch latest rate, this is a goroutine
```
Comment (dreacot) [inline ui/page/accounts/accounts_page.go:119]: this should have been true yeah? @itswisdomagain

File: ui/page/root/wallet_selector_page.go [hunk 3]
```diff
@@ -138,7 +138,7 @@ func (pg *WalletSelectorPage) OnNavigatedTo() {
 				break
 			}
 
-			rate := pg.AssetsManager.RateSource.GetTicker(marketValue)
+			rate := pg.AssetsManager.RateSource.GetTicker(marketValue, false) // okay to fetch latest rate, this is a goroutine
```
Comment (dreacot) [inline ui/page/root/wallet_selector_page.go:141]: this should have been true too yeah?

File: ui/page/send/page.go [hunk 4]
```diff
@@ -282,7 +282,7 @@ func (pg *Page) fetchExchangeRate() {
 		return
 	}
 
-	rate := pg.AssetsManager.RateSource.GetTicker(market)
+	rate := pg.AssetsManager.RateSource.GetTicker(market, false) // okay to fetch latest rate, this is a goroutine
```
Comment (dreacot) [inline ui/page/send/page.go:285]: and this?

File: ui/page/wallet/single_wallet_main_page.go [hunk 5]
```diff
@@ -230,7 +230,7 @@ func (swmp *SingleWalletMasterPage) fetchExchangeRate() {
 		return
 	}
 
-	rate := swmp.AssetsManager.RateSource.GetTicker(market)
+	rate := swmp.AssetsManager.RateSource.GetTicker(market, false) // okay to fetch latest rate, this is a goroutine
```
Comment (dreacot) [inline ui/page/wallet/single_wallet_main_page.go:233]: and this?

File: ui/page/accounts/accounts_page.go (hunk 2 above)
Comment (ukane-philemon) [reply ui/page/accounts/accounts_page.go:119]: Nah, this method is called in a goroutine and network calls won't hang forever.

**Pull request merged**